fractal.sdk/src/test/java/com/yanchware/fractal/sdk/domain/entities/livesystem/paas/providers/azure/appservice/AzureDeploymentSlotTest.java

```diff
@@ -0,0 +1,43 @@
+package com.yanchware.fractal.sdk.domain.entities.livesystem.paas.providers.azure.appservice;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class AzureWebAppDeploymentSlotTest {
+  @Test
+  void builder_withCloneSettingsFromWebApp_createsExpectedDeploymentSlot() {
+    // Setup
+    boolean expectedCloneSettings = true; // Assuming the default or specific scenario you want to test
+
+    // Act
+    var deploymentSlot = AzureWebAppDeploymentSlot.builder()
```

Comment:
Why is this an independent components?
This to me makes no sense. A deployment slot is an extension of an AzureAppService functionality. It should not be an independent component.